scripts.py

```diff
@@ -433,27 +428,16 @@ def process_lane(self, lane):
         return cached_feeds
         
 class CacheFacetListsPerLane(CacheRepresentationPerLane):
-    """Cache the first two pages of every facet list for this lane."""
+    """Cache the first two pages of every relevant facet list for this lane."""
 
     name = "Cache OPDS feeds"
-
-    @classmethod
-    def facet_settings(cls, library, group_name):
-        enabled = library.enabled_facets(group_name)
-        default = library.default_facet(group_name)
-        return enabled, default
     
     @classmethod
     def arg_parser(cls, _db):
         parser = CacheRepresentationPerLane.arg_parser(_db)
-
-        library = Library.instance(_db)
-        
-        enabled, default = cls.facet_settings(
-            library, Facets.ORDER_FACET_GROUP_NAME
-        )
-        order_help = 'Generate feeds for this ordering. Possible values: %s. Default: %s' % (
-            ", ".join(enabled), default
+        available = Facets.DEFAULT_ENABLED_FACETS[Facets.ORDER_FACET_GROUP_NAME]
+        order_help = 'Generate feeds for this ordering. Possible values: %s.' % (
+            ", ".join(available)
```

Comment:
We no longer know which facets are available, since it's a per-library setting and this script runs against all libraries.
Showing the general default list is the best we can do.